exposed-java-time/src/main/kotlin/org/jetbrains/exposed/sql/javatime/JavaDateFunctions.kt

```diff
@@ -65,25 +63,31 @@ object CurrentDate : Function<LocalDate>(JavaLocalDateColumnType.INSTANCE) {
 }
 
 /**
- * Represents an SQL function that returns the current date and time, as [LocalDateTime].
+ * Represents an SQL function that returns the current date and time, as [LocalDateTime] with the specified fractional seconds [precision].
  *
  * @sample org.jetbrains.exposed.DefaultsTest.testConsistentSchemeWithFunctionAsDefaultExpression
  */
-object CurrentDateTime : CurrentTimestampBase<LocalDateTime>(JavaLocalDateTimeColumnType.INSTANCE)
+open class CurrentDateTime(precision: Byte? = null) : CurrentTimestampBase<LocalDateTime>(precision, JavaLocalDateTimeColumnType(precision)) {
+    companion object : CurrentDateTime()
+}
 
 /**
- * Represents an SQL function that returns the current date and time, as [Instant].
+ * Represents an SQL function that returns the current date and time, as [Instant]  with the specified fractional seconds [precision].
  *
  * @sample org.jetbrains.exposed.DefaultsTest.testConsistentSchemeWithFunctionAsDefaultExpression
  */
-object CurrentTimestamp : CurrentTimestampBase<Instant>(JavaInstantColumnType.INSTANCE)
+open class CurrentTimestamp(precision: Byte? = null) : CurrentTimestampBase<Instant>(precision, JavaInstantColumnType(precision)) {
+    companion object : CurrentTimestamp()
+}
 
 /**
- * Represents an SQL function that returns the current date and time with time zone, as [OffsetDateTime].
+ * Represents an SQL function that returns the current date and time with time zone, as [OffsetDateTime] with the specified fractional seconds [precision].
  *
  * @sample org.jetbrains.exposed.DefaultsTest.testTimestampWithTimeZoneDefaults
  */
-object CurrentTimestampWithTimeZone : CurrentTimestampBase<OffsetDateTime>(JavaOffsetDateTimeColumnType.INSTANCE)
+open class CurrentTimestampWithTimeZone(precision: Byte? = null) : CurrentTimestampBase<OffsetDateTime>(precision, JavaOffsetDateTimeColumnType(precision)) {
+    companion object : CurrentTimestampWithTimeZone()
```

Comment:
After changing `CurrentTimestampWithTimeZone` from an object to a class, this line enables users to still use it like this `CurrentTimestampWithTimeZone` and not be forced to use it like this `CurrentTimestampWithTimeZone()`, thereby avoiding a breaking change.